client/components/productCardCart.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import {Link} from 'react-router-dom'
+import {connect} from 'react-redux'
+import {deleteItem, updateItem} from '../store/cart/actions'
+
+const ProductCardCart = ({orderProduct, deleteFromCart, updateCart}) => {
+  const product = orderProduct.product
+  if (product) {
+    return (
+      <div className="productCardContainer">
+        <div className="imgContainer">
+          <div
+            className={
+              product.bestSeller
+                ? 'bestSeller bestSellerShow'
+                : 'bestSeller bestSellerNotShow'
+            }
+          >
+            Best Seller
+          </div>
+          <img
+            src={product.imageURL}
```

Comment:
After I merged my images S3 PR, we no longer have this field in the products table.
So, `product.images` is an array of image objects. Each image object has a `url` field that links to the image itself hosted on S3.

Reply:
Ah got it, fixed up